src/phy/sys/raw_socket.rs

```diff
@@ -68,7 +68,7 @@ impl RawSocketDesc {
         unsafe {
             let len = libc::send(self.lower, buffer.as_ptr() as *const libc::c_void,
                                  buffer.len(), 0);
-            if len == -1 { Err(io::Error::last_os_error()).unwrap() }
+            if len == -1 { return Err(io::Error::last_os_error()) }
```

Comment:
Why this change? The unwrap was deliberate.

Reply:
Could you please elaborate why you should not be able to gracefully handle the error?

Reply:
IIRC, my reasoning was that this call should never fail.